### DISCUSSION TITLE
py-memory setting

### DISCUSSION THREAD

**Discussion by ArturKhuziakhmetov** [pymupdf/PyMuPDF#1691] [Q&A] [answered]
My configuration:
Python 3.10.3 (tags/v3.10.3:a342a49, Mar 16 2022, 13:07:40) [MSC v.1929 64 bit (AMD64)] on win32
>>> fitz.TOOLS.fitz_config
{'plotter-g': True, 'plotter-rgb': True, 'plotter-cmyk': True, 'plotter-n': True, 'pdf': True, 'xps': True, 'svg': True, 'cbz': True, 'img': True, 'html': True, 'epub': True, 'jpx': True, 'js': True, 'tofu': False, 'tofu-cjk': True, 'tofu-cjk-ext': False, 'tofu-cjk-lang': False, 'tofu-emoji': False, 'tofu-historic': False, 'tofu-symbol': False, 'tofu-sil': True, 'icc': True, 'base14': True, **'py-memory': False**}
Also i see #define JM_MEMORY 0 in file fitz.i.
But docs say: 'We are using option this for Python version 3 since PyMuPDF v1.13.19.' So I think, py-memory should be True.

**Comment by JorjMcKie** [comment, marked as answer]
No, I have currently disabled this feature.
I am having suspicions that in the most recent MuPDF version(s) things are not really stable.
I will reactivate again eventually.

Thanks for bringing this up, though.
